google/cloud/aiplatform/metadata/metadata.py

```diff
@@ -48,8 +57,12 @@ def experiment_name(self) -> Optional[str]:
     @property
     def run_name(self) -> Optional[str]:
         """Return the run name of the _MetadataService, if run is not set, return None"""
-        if self._run:
-            return self._run.display_name
+        if _EXPERIMENT_TRACKING_VERSION == "v2":
+            if self._experiment_run:
```

Comment:
`None` because `self._experiment_run` would be `None` if it's not set:

https://github.com/googleapis/python-aiplatform/pull/922/files/8e0e42fce5f89dcf54a233ca1b2359a3d7477f76#diff-06f3b5a184319cfffa5ed8226a331ef9f20736e3204bf9b05fdee5bfd4e821b4R205

https://github.com/googleapis/python-aiplatform/pull/922/files/8e0e42fce5f89dcf54a233ca1b2359a3d7477f76#diff-06f3b5a184319cfffa5ed8226a331ef9f20736e3204bf9b05fdee5bfd4e821b4R47